2021-11-26 23:12:30,892 INFO    MainThread:4816 [wandb_setup.py:_flush():71] setting env: {'api_key': '0f579240b99e6ed291924359c5de323258326760'}
2021-11-26 23:12:30,892 INFO    MainThread:4816 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-26 23:12:30,892 INFO    MainThread:4816 [wandb_init.py:_log_setup():357] Logging user logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211126_231230-9fhtm9oe\logs\debug.log
2021-11-26 23:12:30,892 INFO    MainThread:4816 [wandb_init.py:_log_setup():358] Logging internal logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211126_231230-9fhtm9oe\logs\debug-internal.log
2021-11-26 23:12:30,892 INFO    MainThread:4816 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000286E6391970>
2021-11-26 23:12:30,893 INFO    MainThread:4816 [wandb_init.py:init():390] calling init triggers
2021-11-26 23:12:30,893 INFO    MainThread:4816 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-11-26 23:12:30,893 INFO    MainThread:4816 [wandb_init.py:init():435] starting backend
2021-11-26 23:12:30,893 INFO    MainThread:4816 [backend.py:_multiprocessing_setup():94] multiprocessing start_methods=spawn, using: spawn
2021-11-26 23:12:30,896 INFO    MainThread:4816 [backend.py:ensure_launched():198] starting backend process...
2021-11-26 23:12:30,955 INFO    MainThread:4816 [backend.py:ensure_launched():203] started backend process with pid: 14064
2021-11-26 23:12:30,955 INFO    MainThread:4816 [wandb_init.py:init():444] backend started and connected
2021-11-26 23:12:30,965 INFO    MainThread:4816 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-11-26 23:12:30,965 INFO    MainThread:4816 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-26 23:12:30,965 INFO    MainThread:4816 [wandb_init.py:init():503] updated telemetry
2021-11-26 23:12:30,997 INFO    MainThread:4816 [wandb_init.py:init():533] communicating current version
2021-11-26 23:12:32,335 INFO    MainThread:4816 [wandb_init.py:init():538] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-26 23:12:32,336 INFO    MainThread:4816 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-26 23:12:32,693 INFO    MainThread:4816 [wandb_init.py:init():576] starting run threads in backend
2021-11-26 23:12:33,742 INFO    MainThread:4816 [wandb_run.py:_console_start():1693] atexit reg
2021-11-26 23:12:33,743 INFO    MainThread:4816 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-11-26 23:12:33,743 INFO    MainThread:4816 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-11-26 23:12:33,748 INFO    MainThread:4816 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-26 23:12:33,748 INFO    MainThread:4816 [wandb_init.py:init():603] run started, returning control to user process
2021-11-26 23:12:33,769 INFO    MainThread:4816 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-26 23:12:33,769 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:13:23,745 INFO    MainThread:4816 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-26 23:13:23,746 INFO    MainThread:4816 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-26 23:13:23,746 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:13:24,989 INFO    MainThread:4816 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-26 23:13:24,990 INFO    MainThread:4816 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-26 23:13:24,990 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:13:25,532 INFO    MainThread:4816 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-26 23:13:25,559 INFO    MainThread:4816 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-26 23:13:25,560 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:13:27,263 INFO    MainThread:4816 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-26 23:13:27,712 INFO    MainThread:4816 [wandb_setup.py:_flush():71] setting env: {'api_key': '0f579240b99e6ed291924359c5de323258326760'}
2021-11-26 23:13:27,713 INFO    MainThread:4816 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-26 23:13:27,713 INFO    MainThread:4816 [wandb_init.py:_log_setup():357] Logging user logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211126_231327-2ljn57fc\logs\debug.log
2021-11-26 23:13:27,714 INFO    MainThread:4816 [wandb_init.py:_log_setup():358] Logging internal logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211126_231327-2ljn57fc\logs\debug-internal.log
2021-11-26 23:13:27,714 INFO    MainThread:4816 [wandb_init.py:init():390] calling init triggers
2021-11-26 23:13:27,714 INFO    MainThread:4816 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-11-26 23:13:27,714 INFO    MainThread:4816 [wandb_init.py:init():410] re-initializing run, found existing run on stack: 9fhtm9oe
2021-11-26 23:13:27,714 INFO    MainThread:4816 [wandb_run.py:finish():1347] finishing run rahav-manoharan/demo/9fhtm9oe
2021-11-26 23:13:27,714 INFO    MainThread:4816 [jupyter.py:save_history():429] not saving jupyter history
2021-11-26 23:13:27,714 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:13:27,714 INFO    MainThread:4816 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-11-26 23:13:27,714 INFO    MainThread:4816 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-26 23:13:27,714 INFO    MainThread:4816 [wandb_run.py:_restore():1635] restore
2021-11-26 23:13:28,722 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 1187
}

2021-11-26 23:13:29,348 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 1187
}

2021-11-26 23:13:29,685 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 1187
}

2021-11-26 23:13:30,125 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 4814
}

2021-11-26 23:13:30,227 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 4814
}

2021-11-26 23:13:30,335 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 4814
}

2021-11-26 23:13:30,446 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 4814
}

2021-11-26 23:13:30,555 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 4814
}

2021-11-26 23:13:30,663 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4814
  total_bytes: 4814
}

2021-11-26 23:13:30,765 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4814
  total_bytes: 4814
}

2021-11-26 23:13:30,876 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4814
  total_bytes: 4814
}

2021-11-26 23:13:30,985 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4814
  total_bytes: 4814
}

2021-11-26 23:13:31,095 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4814
  total_bytes: 4814
}

2021-11-26 23:13:31,202 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4814
  total_bytes: 4814
}

2021-11-26 23:13:31,311 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4814
  total_bytes: 4814
}

2021-11-26 23:13:31,873 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4814
  total_bytes: 4814
}

2021-11-26 23:13:32,567 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4814
  total_bytes: 4814
}
local_info {
}

2021-11-26 23:13:33,657 INFO    MainThread:4816 [wandb_run.py:_append_history():2011] rendering history
2021-11-26 23:13:33,658 INFO    MainThread:4816 [wandb_run.py:_append_summary():1966] rendering summary
2021-11-26 23:13:33,658 INFO    MainThread:4816 [wandb_run.py:_append_files():2061] logging synced files
